poc/pkg/templates/deployment.yaml

```diff
@@ -47,6 +47,17 @@ spec:
               port: http
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
```

Comment:
We don't need to mount the secret in the main deployment flie, so we can get rid of this.